src/buildercore/terraform.py

```diff
@@ -1454,20 +1486,78 @@ def init(stackname, context):
             },
         }
 
+    backend = {
+        'terraform': {
+            # 'required_providers': {
+            #    'fastly': {
+            #        'source': 'fastly/fastly',
+            #    }
+            # },
+            'backend': {
```

Comment:
add a comment explaining that this is how terraform state is managed, separately from how aws state is managed